File: absp1.py
from itertools import combinations
cases = int(input())
for a in range(cases):
    s = input()
    x = combinations(map(int,raw_input().split()),2)
    sum = 0 
    for d in x:  
        sum += abs(d[0]-d[1]) 
    print (sum)

File: absp1.c
#include <stdio.h>
#include <stdlib.h>
int main(){
    int c;
    scanf("%d", &c);
    while (c--){
        int s;
        int sum = 0;
        scanf("%d", &s);
        int r[s];
        int x;
        for (x=0;x<s;++x){
            int t;
            scanf("%d", &t);
            r[x] = t;
        }
        for (x=0;x<s;++x){
            int t;
            for(t=(x+1);t<s;++t){
                sum += abs(r[x] - r[t]);
            }
        }
        printf("%d\n", sum );
    }
    return 0;
}